user_manual/groupware/mail.rst

```diff
@@ -11,11 +11,13 @@ Managing your mail account
 ---------------------------
 
 Switch layout
-~~~~~~~~~~~~~
+~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
this line has to be as long as the heading. you should see a warning in the sphinx build log